build.rs

```diff
@@ -159,7 +162,8 @@ const RING_INCLUDES: &'static [&'static str] =
       "include/GFp/bn.h",
       "include/GFp/cpu.h",
       "include/GFp/mem.h",
-      "include/GFp/type_check.h"];
+      "include/GFp/type_check.h",
+      "include/versioned.h"];
```

Comment:
This file should be under a `GFp` directory so that all the include statements are of the form `GFp/versioned.h` to prevent conflicts. If that's problematic then we could also just rename the file to have a "gfp_" prefix to its base name.

Reply:
Should `yasm_versioned.h` be here too? The same name collision concerns apply to it.

Reply:
Hmm. Well, we don't know the path to `yasm_versioned.h` since it's generated at build-time, so we wouldn't be able to add it to this array. Because it's generated, however, I don't think it needs to be included in this array. Please do correct me if I'm wrong, however.